iTunesSearch/iTunesSearch/ViewControllers/SearchResultsTableViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  SearchResultsTableViewController.swift
+//  iTunesSearch
+//
+//  Created by Cora Jacobson on 7/8/20.
+//  Copyright © 2020 Cora Jacobson. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultsTableViewController: UITableViewController {
+    
+    let searchResultController = SearchResultController()
+    
+    @IBOutlet weak var searchTypeSegmentedControl: UISegmentedControl!
+    @IBOutlet weak var searchTermSearchBar: UISearchBar!
+    @IBOutlet weak var resultsLimitSlider: UISlider!
+    @IBOutlet weak var resultsLimitLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        searchTermSearchBar.delegate = self
+    }
+
+    @IBAction func searchTypeChanged(_ sender: UISegmentedControl) {
```

Comment:
Awesome job implementing the automatic search when sliding the segmented control.